react/features/recording/components/Recording/native/StartRecordingDialogContent.tsx

```diff
@@ -0,0 +1,339 @@
+/* eslint-disable lines-around-comment  */
+import React from 'react';
+import { Image, View } from 'react-native';
+import { Switch, Text } from 'react-native-paper';
+
+// @ts-ignore
+import { translate } from '../../../../base/i18n';
+// @ts-ignore
+import { LoadingIndicator } from '../../../../base/react';
+// @ts-ignore
+import { connect } from '../../../../base/redux';
+import Button from '../../../../base/ui/components/native/Button';
+import { BUTTON_TYPES } from '../../../../base/ui/constants';
+// @ts-ignore
+import { RECORDING_TYPES } from '../../../constants';
+// @ts-ignore
+import { getRecordingDurationEstimation } from '../../../functions';
+import AbstractStartRecordingDialogContent, {
+    type Props,
+    mapStateToProps
+} from '../AbstractStartRecordingDialogContent';
+import {
+    DROPBOX_LOGO,
+    ICON_CLOUD,
+    ICON_INFO,
+    ICON_USERS,
+    TRACK_COLOR
+    // @ts-ignore
+} from '../styles.native';
+
+
+/**
+ * The start recording dialog content for the mobile application.
+ */
+class StartRecordingDialogContent extends
+// @ts-ignore
```

Comment:
Why is this needed?

Reply:
Type 'AbstractStartRecordingDialogContent' is not generic.

Reply:
That means that you tried to pass a type to it when it doesn't support it. Just remove `<Props>` from `AbstractAbstractStartRecordingDialogContent<Props>` or make the component generic by modifying AbstractStartRecordingDialogContent to receive a type for Props like `class AbstractStartRecordingDialogContent<P extends Props> extends Component<P>`

Reply:
Thanks for the hint!